lib/engine-addon.js

```diff
@@ -372,6 +372,11 @@ module.exports = {
 
       var result = originalInit.apply(this, arguments);
 
+      var useDeprecatedIncorrectCSSProcessing = this.useDeprecatedIncorrectCSSProcessing === true;
+      if (useDeprecatedIncorrectCSSProcessing) {
+        this.ui.writeDeprecateLine(this.pkg.name + ' engine has opted in to ember-engines@0.4 styles processing compatibility mode, this is deprecated and will be removed in future versions.');
```

Comment:
Nit: Should note that "engine has opted in to ember-engines@0.4 styles processing compatibility **via use of the useDeprecatedIncorrectCSSProcessing flag**". Just to make it clear for those that might not be familiar with the option.